Gems/PhysX/Code/Include/PhysX/EditorColliderComponentRequestBus.h

```diff
@@ -82,4 +82,15 @@ namespace PhysX
     };
 
     using EditorColliderComponentRequestBus = AZ::EBus<EditorColliderComponentRequests>;
+
+    class EditorColliderValidationRequests : public AZ::ComponentBus
+    {
+    public:
+        // TODO
```

Comment:
Remove the `TODO` and replace with API comment.

Reply:
+1